File: 8kyu/Max_Min/code.java
public class Kata {

    public int max(int[] list) {
      int max = list[0];
      for (int i : list) {
        if(max<i){max=i;}
      }
      return max;
    }
    
    public int min(int[] list) {
      int min = list[0];
      for(int i : list){
        if(i<min){min=i;}
      }
      return min;
    }
}

File: 8kyu/Max_Min/code.js
var min = function (list) {
  let min = list[0];
  for (let x = 0; x < list.length; x++) {
    min = Math.min(min, list[x]);
  }
  return min;
};

var max = function (list) {
  let max = list[0];
  for (let x = 0; x < list.length; x++) {
    max = Math.max(max, list[x]);
  }
  return max;
};
